packages/odyssey-react/src/components/Toast/index.tsx

```diff
@@ -42,14 +45,14 @@ export interface PropsToast {
   
   onAnimationEnd?: (event: AnimationEvent) => void,
 
-  onDismiss: () => void
+  onDismiss?: ButtonProps['onClick']
 }
 
 export type PropsToastProvider = { 
   /**
    * Child react nodes which leverage the toast context. This is typically an entire app. 
    */
-  children?: ReactElement | ReactElement[],
+  children?: ReactNode | ReactNode[],
```

Comment:
[incidentally `ReactNode` already takes care of the multiple children use case for us!](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/1349b640d4d07f40aa7c1c6931f18e3fbf667f3a/types/react/index.d.ts#L239-L241)

So this can simply be `children?: ReactNode`